Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```
@@ -3,1557 +3,1774 @@ commonfields:
+          name: sha256
+          required: true
+          secret: false
+      deprecated: false
+      description: Return a summary of the observed file paths
+      execution: false
+      name: cb-eedr-file-paths
+      outputs:
+        - contextPath: CarbonBlackEEDR.File.file_path_count
+          description: The total number of unique file paths that have been observed,
+            by this organization, for this file.
+          type: Number
+        - contextPath: CarbonBlackEEDR.File.file_paths
+          description: The file path details.
+          type: String
+        - contextPath: CarbonBlackEEDR.File.sha256
+          description: The SHA256 hash of the file.
+          type: Unknown
+        - contextPath: CarbonBlackEEDR.File.total_file_path_count
+          description: The total number of file paths that have been observed, by this
+            organization, for this file.
+          type: Number
+    - arguments:
+        - default: false
+          description: The process name to search.
+          isArray: true
+          name: process_name
+          required: false
+          secret: false
+        - default: false
+          description: The process hash to search.
+          isArray: true
+          name: process_hash
+          required: false
+          secret: false
+        - default: false
+          description: The event id to search.
+          isArray: false
+          name: event_id
+          required: false
+          secret: false
+        - default: false
+          description: The maximum number of rows to return.
+          isArray: false
+          name: limit
+          required: false
+          secret: false
+        - default: false
+          description: A free-style query.
+          isArray: false
+          name: query
+          required: false
+          secret: false
+      deprecated: false
+      description: Creates a process search job.
+      execution: false
+      name: cb-eedr-process-search
+      outputs:
+        - contextPath: CarbonBlackEEDR.SearchProcess.job_id
+          description: A request job id.
+          type: String
+    - arguments:
+        - default: false
+          description: The process guid to search.
+          isArray: false
+          name: process_guid
+          required: false
+          secret: false
+        - default: false
+          description: The event type to search.
+          isArray: false
+          name: event_type
+          required: false
+          secret: false
+        - default: false
+          description: The maximum number of rows to return.
+          isArray: false
+          name: limit
+          required: false
+          secret: false
+        - default: false
+          description: A free-style query.
+          isArray: false
+          name: query
+          required: false
+          secret: false
+      deprecated: false
+      description: Fetch the events associated with a given process.
```

Fetch  is associated with fetch-incidents, best to rename.
```suggestion
      description: Retrieve the events associated with a given process.
```